Examples/Python/src/TGeo.cpp

```diff
@@ -0,0 +1,89 @@
+// This file is part of the ACTS project.
+//
+// Copyright (C) 2016 CERN for the benefit of the ACTS project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "Acts/Plugins/Python/Utilities.hpp"
+#include "Acts/Plugins/TGeo/TGeoDetectorElement.hpp"
+#include "Acts/Plugins/TGeo/TGeoLayerBuilder.hpp"
+#include "Acts/Plugins/TGeo/TGeoParser.hpp"
+
+#include <vector>
+
+#include <TGeoManager.h>
+#include <TGeoVolume.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/stl/filesystem.h>
+
+namespace py = pybind11;
+using namespace pybind11::literals;
+
+namespace Acts::Python {
+void addTGeo(Context& ctx) {
+  auto [m, mex] = ctx.get("main", "examples");
+
+  auto tgeo = mex.def_submodule("tgeo");
+
+  {
+    py::class_<Acts::TGeoDetectorElement,
+               std::shared_ptr<Acts::TGeoDetectorElement>>(
+        tgeo, "TGeoDetectorElement")
+        .def("surface", [](const Acts::TGeoDetectorElement& self) {
+          return self.surface().getSharedPtr();
+        });
+  }
+
+  {
+    /// Helper function to test if the automatic geometry conversion works
+    ///
+    /// @param rootFileName is the name of the GDML file
+    /// @param sensitiveMatches is a list of strings to match sensitive volumes
+    /// @param localAxes is the TGeo->ACTS axis conversion convention
+    /// @param convertMaterial is a flag to convert the material
+    tgeo.def("convertToElements",
```

Comment:
If this is an internal function, maybe `_convertToElements`?

Reply:
Ok, are we doing this consistently now? 

I am happy to change, I've also added some information why this method is useful.

Reply:
Ok, understood, probably we should make some sort of a policy for that then. What I want to achieve here is for people who have a TGeo object in hand to test whether the conversion into detector elements works or not.

Reply:
Fixed.